packages/vite/src/node/plugins/css.ts

```diff
@@ -537,7 +537,7 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
           if (cssModuleRE.test(id)) {
```

Comment:
Do we need to consider the case when `style.has(id) = true` and `!!chunk.modules[id].code = true`?

Reply:
I think that would be unrelated to this PR, but I don't think so. Maybe you could try to create a reproduction for this case and we fixed it in a different PR. For reference, I used `.code` here because it seemed a more robust way to check if the module was an HTML entry (or some non-HTML entry in frameworks?). Open to suggestions about this.

Reply:
I can't figure out  a reproduction for this case yet, then this pr lgtm.